charts/internal/control-plane/values.yaml

```diff
@@ -41,6 +42,15 @@ authnWebhook:
     issuerUrl: https://tokenissuer/dex
     issuerClientId: myClientId
 
+auditlogWebhook:
+  enabled: false
+  port: 443
+  accept-insecure-cert: false
```

Comment:
I think camel case would be a little easier to handle and it's also what the other configuration values are using as a convention.

Reply:
Changed.